obal/data/playbooks/nightly.yml

```diff
@@ -3,9 +3,28 @@
     - packages
   serial: 1
   gather_facts: no
-  roles:
-    - build_package
   vars:
-    releasers:
-      - "{{ nightly_releaser }}"
-    build_package_tito_releaser_args: "{{ nightly_package_tito_releaser_args }}"
+    build_package_tito_releaser_args: "{{ nightly_package_tito_releaser_args | default([]) }}"
+  tasks:
+    - name: 'set nightly_releaser'
+      set_fact:
+        releasers:
+          - "{{ nightly_releaser }}"
+      when: nightly_releaser is defined and "--arg jenkins_job=" in (build_package_tito_releaser_args | join(" "))
```

Comment:
You shouldn't need to do a join here as python should perform an `in` check on an array.

Reply:
It does make sense because now you can do a sort of fuzzy matching. Otherwise you need an exact match which is impossible to know with a job ID.

Reply:
> You shouldn't need to do a join here as python should perform an `in` check on an array.

It's been a week or so since I wrote this, but I'm fairly certain I tried it without `join(" ")` first. Just to double-check, I'll try it again though.

Reply:
> It does make sense because now you can do a sort of fuzzy matching. Otherwise you need an exact match which is impossible to know with a job ID.

Not sure I'm following you. Mind adding more context?

Reply:
Like I said: I think the current code is the easiest way to achieve what you want so I wouldn't bother :)

Reply:
If it's an array `['--arg jenkins_job=1234']` then `'--arg jenkins_job' in x` will fail while it would work with your joined string.